lib/parity/environment.rb

```diff
@@ -104,7 +104,7 @@ def heroku_app_name
     end
 
     def basename
-      Parity.config.heroku_app_basename || Dir.pwd.split('/').last
+      Dir.pwd.split('/').last
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.